cypress/e2e/process.cy.js

```diff
@@ -0,0 +1,163 @@
+            processesPage.textAutomaticChecksFailed().should('be.visible');
+            tenureReviewDocsPage.checkboxConfirmOutcomeLetter().click();
+            tenureReviewDocsPage.buttonConfirm().click();
+            cy.contains('Thank you for your confirmation');
+            cy.contains("This case is now closed and we have recorded this on the system - that you have sent an outcome letter to the resident. The outcome can be viewed in the activity history");
+            cy.url().should("include", "manage-my-home")
+        })
+    })
+
+    it('should fail on manual checks', ()=> {
+        seedDatabase()
+        cy.getTenureFixture().then((tenureInfo) => {
+            processesPage.visit(tenureInfo.id)
+
+            cy.getAssetFixture().then((asset) => {
+                const { assetAddress } = asset;
+                processesPage.tenureDetails().should("be.visible")
+                processesPage.tenureDetails().contains(`Tenure payment ref ${asset.tenure.paymentReference}`)
+                processesPage.tenureDetails().contains(`${assetAddress.addressLine1} ${assetAddress.addressLine2} ${assetAddress.addressLine3} ${assetAddress.postCode}`)
+            })
+
+            processesPage.startProcessButton().should("be.disabled")
+            processesPage.agreementCheckBox().click()
+            processesPage.startProcessButton().click()
+            cy.url().should("include", "processes/soletojoint/")
+            processesPage.headingSoleTenantRequestsAJointTenure().should('be.visible');
+            cy.get(`#select-tenant-${tenureInfo.householdMembers[0].id}`).click({ force: true })
+            cy.contains("Next").click();
+            processesPage.textAutomaticEligibilityChecksPassed().should('be.visible');
+
+            tenureReqDocsPage.selectYesFor12Months().click();
+            tenureReqDocsPage.selectNoForOccupyanyOther().click();
+            tenureReqDocsPage.selectYesForSurvivorOfOne().click();
+            tenureReqDocsPage.selectYesForTenantEvicted().click();
+            tenureReqDocsPage.selectYesForImmigrationControl().click();
+            tenureReqDocsPage.selectYesForLiveNotice().click();
+            tenureReqDocsPage.selectYesForRentArrears().click();
+            tenureReqDocsPage.selectNoForHoldATenancyElseWhere().click();
+
+            cy.contains("Next").click();
+            processesPage.textAutomaticEligibilityChecksPassed().should('be.visible');
+            processesPage.textAutomaticChecksFailed().should('be.visible');
+            tenureReviewDocsPage.checkboxConfirmOutcomeLetter().click();
+            tenureReviewDocsPage.buttonConfirm().click();
+            cy.contains('Thank you for your confirmation');
+            cy.contains("This case is now closed and we have recorded this on the system - that you have sent an outcome letter to the resident. The outcome can be viewed in the activity history");
+            cy.url().should("include", "manage-my-home")

```

Comment:
I'm not sure this tests the same thing that the `href` intended.

I think the goal of that assertion was to test where that link leads to is the expected URL.
Without having clicked that `a` link, the `cy.url()` does not test the URL behind the `a` link for which the assertion was removed.

When I was referring to testing its `href` value, I was thinking something likes:
```js
cy
  .get('a')
  .invoke('attr', 'href')
  .should('eq', 'https://docs.cypress.io')
```